### ISSUE TITLE
Cannot run binary due to a type error

### ISSUE THREAD

**Issue by hanbyul-kim** [ttscoff/howzit#1]
Hi!
I tried to install `howzit` but I couldn't succeed. Do you have any idea on this?

```shell
Traceback (most recent call last):
	6: from /usr/local/bin/howzit:23:in `<main>'
	5: from /usr/local/bin/howzit:23:in `load'
	4: from /var/lib/gems/2.7.0/gems/howzit-2.0.31/bin/howzit:13:in `<top (required)>'
	3: from /var/lib/gems/2.7.0/gems/howzit-2.0.31/bin/howzit:13:in `new'
	2: from /usr/lib/ruby/2.7.0/optparse.rb:1093:in `initialize'
	1: from /var/lib/gems/2.7.0/gems/howzit-2.0.31/bin/howzit:144:in `block in <top (required)>'
/var/lib/gems/2.7.0/gems/howzit-2.0.31/bin/howzit:144:in `basename': no implicit conversion of nil into String (TypeError)
```

**Comment by ttscoff** [comment]
This error references an attempt to load a default editor. What was the command you used to cause this error? And do you have a config file set up already? It should have created one for you in ~/.config/howzit.

**Comment by kjaymiller** [comment]
I'm running into the same issue.

- installation was done using `brew gem install howzit`
- error comes up when running `howzit build` with no custom configuration.

Device Platform Info:
Software:

    System Software Overview:

      System Version: macOS 14.5 (23F79)
      Kernel Version: Darwin 23.5.0
      Boot Volume: Macintosh HD
      Boot Mode: Normal
      Secure Virtual Memory: Enabled
      System Integrity Protection: Enabled

Hardware:

    Hardware Overview:

      Model Name: MacBook Pro
      Model Identifier: MacBookPro18,1
      Model Number: Z14W00105LL/A
      Chip: Apple M1 Pro
      Total Number of Cores: 10 (8 performance and 2 efficiency)
      Memory: 32 GB
      System Firmware Version: 10151.121.1
      OS Loader Version: 10151.121.1
      Activation Lock Status: Disabled

**Comment by kjaymiller** [comment]
I was able to fix the issue by adding my my editor of choice into `.config/howzit/howzit.yaml`

My assumption was that since I didn't have `EDITOR` filled out in my `zshrc` it didn't populate the information. 

perhaps a suggestion would be to **apply a default** (`nano`, `vim`) or **prompt**. Happy to contribute to the project.

**Comment by ttscoff** [comment]
This should be fixed in 2.1.15. The code changes ran a bit deep, but you can see the [changes here](https://github.com/ttscoff/howzit/commit/a2b6e5cc53ee22dcb519f240309fcf9aca5d97c8) but there's a bunch of test stuff mixed in that's irrelevant.
